room-battle.js

```diff
@@ -258,40 +258,25 @@ class BattleTimer {
 			if (!this.waitingForChoice(slot)) continue;
 			if (!connected && this.dcTicksLeft[slotNum] > 0) {
 				this.dcTicksLeft[slotNum]--;
```

Comment:
Also, when did this become wrong? `ticksLeft` and `turnTicksLeft` should also tick down if `this.settings.dcTimerBank` is off.